### ISSUE TITLE
Physics-based Deep Learning

### ISSUE THREAD

**Issue by guevara** [guevara/read-it-later#11642]
Physics-based Deep Learning<br>
<br>
https://ift.tt/ZeAyEbo<br>
<br>
<br>
<br>

<div><div><div><div><section><h1>Welcome …</h1>
<figure><img alt="_images/logo-xl.jpg" src="https://physicsbaseddeeplearning.org/_images/logo-xl.jpg"></figure><p>Welcome to the <em>Physics-based Deep Learning Book</em> (v0.2) 👋</p>
<p><strong>TL;DR</strong>:
This document contains a practical and comprehensive introduction of everything
related to deep learning in the context of physical simulations.
As much as possible, all topics come with hands-on code examples in the
form of Jupyter notebooks to quickly get started.
Beyond standard <em>supervised</em> learning from data, we’ll look at <em>physical loss</em> constraints,
more tightly coupled learning algorithms with <em>differentiable simulations</em>,
training algorithms tailored to physics problems,
as well as
reinforcement learning and uncertainty modeling.
We live in exciting times: these methods have a huge potential to fundamentally
change what computer simulations can achieve.</p>
<hr>
<section><h2>Coming up</h2>
<p>As a <em>sneak preview</em>, the next chapters will show:</p>
<ul>
<li><p>How to train networks to infer a fluid flow around shapes like airfoils, and estimate the uncertainty of the prediction. This gives a <em>surrogate model</em> that replaces a traditional numerical simulation.</p></li>
<li><p>How to use model equations as residuals to train networks that represent solutions, and how to improve upon these residual constraints by using <em>differentiable simulations</em>.</p></li>
<li><p>How to more tightly interact with a full simulator for <em>inverse problems</em>. E.g., we’ll demonstrate how to circumvent the convergence problems of standard reinforcement learning techniques by leveraging simulators in the training loop.</p></li>
<li><p>We’ll also discuss the importance of <em>inversion</em> for the update steps, and how higher-order information can be used to speed up convergence, and obtain more accurate neural networks.</p></li>
</ul>
<p>Throughout this text,
we will introduce different approaches for introducing physical models
into deep learning, i.e., <em>physics-based deep learning</em> (PBDL) approaches.
These algorithmic variants will be introduced in order of increasing
tightness of the integration, and the pros and cons of the different approaches
will be discussed. It’s important to know in which scenarios each of the
different techniques is particularly useful.</p>
<div>
<p>Executable code, right here, right now</p>
<p>We focus on Jupyter notebooks, a key advantage of which is that all code examples
can be executed <em>on the spot</em>, from your browser. You can modify things and
immediately see what happens – give it a try by
<a href="https://colab.research.google.com/github/tum-pbs/pbdl-book/blob/main/intro-teaser.ipynb">[running this teaser example in your browser]</a>.</p>
<p>Plus, Jupyter notebooks are great because they’re a form of <a href="https://en.wikipedia.org/wiki/Literate_programming">literate programming</a>.</p>
</div>
</section><section><h2>Thanks!</h2>
<p>This project would not have been possible without the help of many people who contributed. Thanks to everyone 🙏 Here’s an alphabetical list:</p>
<p>Additional thanks go to
Georg Kohl for the nice divider images (cf. <span>[<a title="Georg Kohl, Kiwon Um, and Nils Thuerey. Learning similarity metrics for numerical simulations. International Conference on Machine Learning, 2020. URL: https://ge.in.tum.de/publications/2020-lsim-kohl/." href="https://physicsbaseddeeplearning.org/references.html#id8">KUT20</a>]</span>),
Li-Wei Chen for the airfoil data image,
and to
Chloe Paillard for proofreading parts of the document.</p>
</section><section><h2>Citation</h2>
<p>If you find this book useful, please cite it via:</p>
<div>
<div><pre><span></span><span>@book</span><span>{</span><span>thuerey2021pbdl</span><span>,</span>
  <span>title</span><span>=</span><span>{</span><span>Physics</span><span>-</span><span>based</span> <span>Deep</span> <span>Learning</span><span>},</span>
  <span>author</span><span>=</span><span>{</span><span>Nils</span> <span>Thuerey</span> <span>and</span> <span>Philipp</span> <span>Holl</span> <span>and</span> <span>Maximilian</span> <span>Mueller</span> <span>and</span> <span>Patrick</span> <span>Schnell</span> <span>and</span> <span>Felix</span> <span>Trost</span> <span>and</span> <span>Kiwon</span> <span>Um</span><span>},</span>
  <span>url</span><span>=</span><span>{</span><span>https</span><span>:</span><span>//</span><span>physicsbaseddeeplearning</span><span>.</span><span>org</span><span>},</span>
  <span>year</span><span>=</span><span>{</span><span>2021</span><span>},</span>
  <span>publisher</span><span>=</span><span>{</span><span>WWW</span><span>}</span>
<span>}</span>
</pre></div>
</div>
</section><p>
</p>
<p>
</p>
<p>
</p>
<p>
</p>
<p>
</p>
<p>
</p>
<p>
</p>
<p>
</p>
<p>
</p>
</section></div></div></div></div><br>
<br>
<br>
<br>
<br>
<br>
via  <br>
<br>
July 12, 2024 at 05:21PM
